### DISCUSSION TITLE
Prisma support for server less env

### DISCUSSION THREAD

**Discussion by naeem-qv** [prisma/prisma#24500] [Q&A]
Will there be support for server less environment dbs like vercel Postgres or supabase?
What I meant by the question is the latest version of prisma still sucks at cold start... 😢

**Comment by janpio** [comment]
Prisma has supported talking to Vercel Postgres and Supabase databases for ages.

In serverless environments you can use the[ normal database drivers](https://www.prisma.io/docs/orm/overview/databases/database-drivers#default-built-in-drivers), or when available, also specific serverless drivers via [Prisma's driver adapters](https://www.prisma.io/docs/orm/overview/databases/database-drivers#driver-adapters). For Vercel Postgres for example you can use the Neon serverless driver: https://www.prisma.io/docs/orm/overview/databases/neon#how-to-use-neons-serverless-driver-with-prisma-orm-preview (Vercel Postgres is Neon database under the hood)

For Supabase there unfortunately is no serverless driver available, so we can not offer it. You can only talk to that database directly. (The Supabase JS Client SDK is not powerful to support the APIs of Prisma)

For edge functions deployments like Vercel Edge Functions or Cloudflare Workers you even _have_ to use our driver adapters with a serverless driver, as direct connections to databases are technically not possible.


> What I meant by the question is the latest version of prisma still sucks at cold start... 😢

We are not aware of that. If you have more concrete feedback and a reproduction, that would be appreciated.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
